apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -279,19 +279,13 @@ export const onSaveFail =
             getState().aichat.savedAiCustomizations,
             getState().aichat.currentAiCustomizations
           );
-          let flaggedProperties;
-          if (
-            changedProperties.includes('systemPrompt') &&
+          const flaggedProperties =
+            changedProperties.includes('systemPrompt') ||
             changedProperties.includes('retrievalContexts')
-          ) {
-            flaggedProperties = 'system prompt and/or retrieval contexts';
-          } else if (changedProperties.includes('systemPrompt')) {
-            flaggedProperties = 'system prompt';
-          } else if (changedProperties.includes('retrievalContexts')) {
-            flaggedProperties = 'retrieval contexts';
-          }
+              ? 'system prompt and/or retrieval contexts'
+              : undefined;
           if (body?.details?.profaneWords?.length > 0 && flaggedProperties) {
-            errorMessage = `Profanity detected in the ${flaggedProperties} and cannot be updated. Please try again.`;
+            errorMessage = `The ${flaggedProperties} have been flagged by our content moderation policy. Please try a different model customization.`;
```

Comment:
@sanchitmalhotra126 - updated this message to be consistent with other messages for flagged content. Checked in [this Slack thread](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1726000611268619).

Reply:
Note that before update, the message was determined by whether only system prompt, only retrievals or both were updated. Because the message is a little more general, I opted to just prompt user to update model customization (system prompt and/or retrievals).